shared/js/utils.js

```diff
@@ -172,6 +172,49 @@ if (inChrome) {
     callback(!!localStorage.getItem(prefName));
   }
 
+  /**
+   * Used for getting a String preference. It will either use the browser
+   * preferences or try to get them from localStorage.
+   *
+   * @param {String}   prefName The name of the preference. Note that mozLoop
+   *                            adds 'loop.' to the start of the string.
+   * @param {Function} callback Invoked with the preference value.  null is
+   *                            returned if no such preference is found.
+   */
+  function getStringPreference(prefName, callback) {
```

Comment:
Not sure we need this function, when we already check from `localStorage` when we use `loop.request("GetLoopPref", pref)` from the standalone side ([standaloneMozLoop](https://github.com/mozilla/loop/blob/master/standalone/content/js/standaloneMozLoop.js#L310-L313))